samples/peripheral/802154_phy_test/Kconfig

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "PHY Test Tool configuration"
+
+choice
+	prompt "Device Mode"
+	default BOTH
+	help
+	  Select the device mode
+
+config DUT_MODE
+	bool "Configure device as a DUT"
+
+config CMD_MODE
+	bool "Configure device as a CMD"
+
+config BOTH
+	bool "Supports both CMD and DUT modes"
+
+endchoice
+
+config POWER
+	int "Transmit Power (dBm)"
+	default -20
+
+config CHANNEL_MASK
+	hex "Channel page and channel number mask"
+	default 0x00000800
+
+config SW_VERSION
+	int "Software Version"
```

Comment:
If I remember correct, allowed values are uint8_t. Perhaps range 0-255 could be used here to limit potential mistakes